app/products/calls/components/emoji_pill.tsx

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {Text, View} from 'react-native';
+
+import Emoji from '@components/emoji';
+import {useTheme} from '@context/theme';
+import {makeStyleSheetFromTheme} from '@utils/theme';
+import {typography} from '@utils/typography';
+
+const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => (
+    {
+        pill: {
+            alignItems: 'center',
+            justifyContent: 'center',
+            flexDirection: 'row',
+            backgroundColor: 'rgba(255,255,255,0.16)',
+            borderRadius: 20,
+            height: 30,
+            paddingLeft: 16,
+            paddingRight: 16,
+            marginLeft: 6,
+        },
+        count: {
+            ...typography('Body', 75, 'SemiBold'),
+            color: theme.buttonColor,
```

Comment:
The background for the call screen is always black, does it make sense to use this themed? What if my custom theme has the button color as black

Reply:
Good catch, thanks!